app/mailers/reviewer_mailer.rb

```diff
@@ -1,6 +1,10 @@
 class ReviewerMailer < ApplicationMailer
-  def completion_email(curation_concern_id)
+  def notify(curation_concern_id, state)
     @curation_concern = ActiveFedora::Base.find(curation_concern_id)
-    mail(to: Plum.config[:email][:reviewer_address], subject: "[plum] #{@curation_concern.human_readable_type} #{@curation_concern.id} is complete")
+    @state = state
+
+    Role.where(name: "notify_#{state}").first_or_create.users.each do |u|
+      mail(to: "#{u.email}", subject: "[plum] #{@curation_concern.human_readable_type} #{curation_concern_id}: #{state}")
```

Comment:
Having been on these kinds of list before, it's definitely better if I can see the other people who received the message, so that I can start a reply-all thread if I need to.